copis/gui/panels/controller.py

```diff
@@ -65,6 +70,44 @@ def on_device_deselected(self) -> None:
         self.update_machine_pos(Point5())
         self.Disable()
 
+    def on_jog_button(self, event: wx.CommandEvent) -> None:
+        """On EVT_BUTTONs, step value accordingly."""
+        button = event.EventObject
+
+        if button.Name[0] in 'xyzns':
+            step = self.xyz_step_ctrl.Value
+        else: # button.Name in 'pt':
+            step = self.pt_step_ctrl.Value
+
+        step = float(step.split(' ')[0])
+        feed_rate = float(str(self.feed_rate_ctrl.Value).split(' ')[0])
```

Comment:
My FancyTextCtrl has a nice `num_value` attribute so you don't have to do any string manipulation. Can you use that instead of `Value` here?

Reply:
oo nice!